modules/renamer/lua/list.lua

```diff
@@ -0,0 +1,814 @@
+        { 17216224, "736" }, -- Riftworn Pyxis
+        { 17216225, "737" }, -- Riftworn Pyxis
+        { 17216226, "738" }, -- Riftworn Pyxis
+        { 17216227, "739" }, -- Cavernous Maw
+        { 17216228, "740" }, -- Moogle
+        { 17216229, "741" }, -- ???
+        { 17216230, "742" }, -- ???
+        { 17216231, "743" }, -- ???
+        { 17216232, "744" }, -- ???
+        { 17216233, "745" }, -- ???
+        { 17216234, "746" }, -- blank
+        { 17216235, "747" }, -- Moogle
+        { 17216236, "748" }, -- skipped
+        { 17216237, "749" }, -- skipped
+        { 17216238, "750" }, -- skipped
+        { 17216239, "751" }, -- skipped
+        { 17216240, "752" }, -- skipped
+        { 17216241, "753" }, -- skipped
+        { 17216242, "754" }, -- skipped
+        { 17216243, "755" }, -- skipped
+        { 17216244, "756" }, -- skipped
+        { 17216245, "757" }, -- skipped
+        { 17216246, "758" }, -- skipped
+        { 17216247, "759" }, -- skipped
+        { 17216248, "760" }, -- skipped
+        { 17216249, "761" }, -- skipped
+        { 17216250, "762" }, -- skipped
+        { 17216251, "763" }, -- skipped
+        { 17216252, "764" }, -- skipped
+        { 17216253, "765" }, -- skipped
+        { 17216254, "766" }, -- skipped
+        { 17216255, "767" }, -- skipped
+        { 17216256, "768" }, -- skipped
+        { 17216257, "769" }, -- skipped
+        { 17216258, "770" }, -- skipped
+        { 17216259, "771" }, -- skipped
+        { 17216260, "772" }, -- skipped
+        { 17216261, "773" }, -- skipped
+        { 17216262, "774" }, -- skipped
+        { 17216263, "775" }, -- skipped
+        { 17216264, "776" }, -- skipped
+        { 17216265, "777" }, -- skipped
+        { 17216266, "778" }, -- blank
+        { 17216267, "779" }, -- Fish Eyes
+        { 17216268, "780" }, -- skipped
+        { 17216269, "781" }, -- Moogle
+        { 17216270, "782" }, -- Moogle
+        { 17216271, "783" }, -- Moogle
+        { 17216272, "784" }, -- ???
+        { 17216273, "785" }, -- Magivore Ternion
+        { 17216274, "786" }, -- Wandering Cloud
+        { 17216275, "787" }, -- Smile Helper
+        { 17216276, "788" }, -- Smile Helper
+        { 17216277, "789" }, -- Smile Helper
+        { 17216278, "790" }, -- Smile Helper
+        { 17216279, "791" }, -- Smile Helper
+        { 17216280, "792" }, -- Smile Helper
+        { 17216281, "793" }, -- Smile Helper
+        { 17216282, "794" }, -- Smile Helper
+        { 17216283, "795" }, -- Smile Helper
+        { 17216284, "796" }, -- Smile Helper
+        { 17216285, "797" }, -- Debug
+        { 17216286, "798" }, -- blank
+        { 17216287, "799" }, -- Slime Tarou
+        { 17216288, "800" }, -- blank
+        { 17216289, "801" }, -- blank
+        { 17216290, "802" }, -- Orgis
+        { 17216291, "803" }, -- blank
+        { 17216292, "804" }, -- ???
+        { 17216293, "805" }, -- blank
+        { 17216511, "1023" },
+    },
+}
```

Comment:
Quite the list for this.  Why the numeric renames?

Reply:
Actually what it was originally made for too - at the time I wasn't sure if ashita's onRender() was fast enough for all 0-1023 dat index slots in a single rendering frame (it was)